### ISSUE TITLE
Testing frameworks do not display a colored output in sbt 1.0.0-RC2

### ISSUE THREAD

**Issue by sjrd** [sbt/sbt#3348]
(See the guidelines for contributing, linked above)

## steps

`build.sbt`
```scala
lazy val foo = project.
  settings(
    libraryDependencies += "com.novocode" % "junit-interface" % "0.11" % "test",
    testOptions in Test ++= Seq(
      Tests.Argument(TestFrameworks.JUnit, "-v", "-a", "-s")
    )
  )
```
`project/build.properties`
```
sbt.version=1.0.0-RC2
```
`foo/src/test/scala/test/SimpleTest.scala`
```scala
package test

import org.junit.Test
import org.junit.Assert._

class SimpleTest {
  @Test def simpleTest(): Unit = {
    assertEquals(42, 30 + 12)
  }
}
```
Start sbt
```
> test
...
[info] Compiling 1 Scala source to /localhome/doeraene/projects/sbt-test/foo/target/scala-2.12/test-classes ...

[info] Done compiling.

[info] Test run started

[info] Test test.SimpleTest.simpleTest started

[debug] Test test.SimpleTest.simpleTest finished, took 0.002 sec

[info] Test run finished: 0 failed, 0 ignored, 1 total, 0.006s

[info] Passed: Total 1, Failed 0, Errors 0, Passed 1

[info] Total time: 1 s, completed Jul 20, 2017 1:54:15 PM
```
without any colors.

In sbt 0.13.16-M1, I get the following output:

![image](https://user-images.githubusercontent.com/535934/28416289-b85422a4-6d53-11e7-9223-e1ee0ddf583f.png)

## problem

In sbt 1.0.0-RC2, there are no colors.

## expectation

I expect the same output as sbt 0.13.x

## notes

sbt version: *1.0.0-RC2*
Possibly related: #3213

**Comment by Duhemm** [comment]
Possibly related too: https://github.com/sbt/sbt/issues/3336 (fix: sbt/util#101)

**Comment by eed3si9n** [comment]
See the screenshot on https://github.com/sbt/util/pull/104.
This will be fixed in next RC.
